apps/bmd/src/pages/InsertCardScreen.tsx

```diff
@@ -35,7 +35,10 @@ const ActivationScreen = ({
   return (
     <Screen flexDirection="row-reverse" white>
       <Sidebar>
-        <ElectionInfo election={election} precinctId={appPrecinctId} />
+        <ElectionInfo
+          electionDefinition={electionDefinition}
+          precinctId={appPrecinctId}
```

Comment:
Add to the hash to the Insert Card screen, per https://github.com/votingworks/bmd/issues/1278
```suggestion
          precinctId={appPrecinctId}
          showElectionHash
```